src/scp/SCP.cpp

```diff
@@ -281,12 +281,20 @@ SCP::getLatestMessage(NodeID const& id)
 }
 
 bool
-SCP::isLatestStatement(SCPStatement const& st)
+SCP::shouldReplaceSt(SCPStatement const& st, SCPStatement const& hint)
```

Comment:
a better name for this method and its arguments would be
`isNewerStatementSubprotocol` or something like that (it's basically `isNewerStatement`)